notebooks/407-person-tracking-webcam/407-person-tracking-paddle.ipynb

```diff
@@ -0,0 +1,915 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "804e0e25-64e0-4292-8540-a20b4206ad38",
+   "metadata": {},
+   "source": [
+    "# Pedestrian counting with OpenVINO™\n",
+    "\n",
+    "This demo shows how to run pedestrian counting inference with OpenVINO, using [PP-YOLOE](https://gitee.com/paddlepaddle/PaddleDetection/blob/release/2.5/configs/ppyoloe) and [OC-SORT](https://gitee.com/paddlepaddle/PaddleDetection/blob/release/2.5/configs/mot/ocsort) from [PaddleDetection Github](https://github.com/PaddlePaddle/PaddleDetection) or [PaddleDetection Gitee](https://gitee.com/paddlepaddle/PaddleDetection). \n",
+    "The PP-YOLOE pre-trained model used in the demo refers to the \"mot_ppyoloe_s_36e_pipeline(27M)\""
```

Comment:
Could you add a diagram the example the workflow of people counting with OC-SORT. It will be helpful for our user to understand the logic behind.